controller.py

```diff
@@ -112,3 +117,181 @@ def remove_items(self):
         )
 
         return feed_response(removal_feed)
+
+
+class URNLookupController(CoreURNLookupController):
+
+    UNRESOLVABLE_IDENTIFIER = "I can't gather information about an identifier of this type."
```

Comment:
This constants already exists in `CoreURNLookupController` (albeit with a slightly different message).